modules/transformations/snowflake/sql/ST_CONCAVEHULL.sql

```diff
@@ -14,10 +14,27 @@ AS $$
 
     @@SF_LIBRARY_CONTENT@@
 
+    const multiPoints = transformationsLib.multiPoint(GEOJSONS.map(x => JSON.parse(x).coordinates));
+    const nonDuplicates = transformationsLib.cleanCoords(multiPoints).geometry;
+    const arrayCoordinates = nonDuplicates.coordinates;
+
+    // Point
+    if (arrayCoordinates.length == 1) {
+        return JSON.stringify(transformationsLib.point(arrayCoordinates[0]).geometry);
+    } 
+
+    // Segment
+    if (arrayCoordinates.length == 2) {
+        const start = arrayCoordinates[0];
+        const end = arrayCoordinates[1];
+        const lineString = transformationsLib.lineString([start, end]);
+        return JSON.stringify(lineString.geometry);
+    }
+
     const options = {};
     options.maxEdge = MAXEDGE;
     options.units = UNITS;
-    const featuresCollection = transformationsLib.featureCollection(GEOJSONS.map(x => transformationsLib.feature(JSON.parse(x))));
+    const featuresCollection = transformationsLib.featureCollection(arrayCoordinates.map(x => transformationsLib.point(x)));
```

Comment:
I guess just after this the `concave` merge all this points into polygons if needed considering the distances.

Reply:
Yes, but that is the reason I'm removing duplicates before, because we can generate a degenerated polygon and turf will throw an error.